content/getting-started/onboarding-wizard.md

```diff
@@ -26,6 +26,12 @@ Cost Explorer must be enabled from the master billing account---even if already
 
 Cost and Usage Reports (CUR) have replaced Detailed Billing in AWS. CloudWisdom recommends switching to (or starting with) a CUR setup instead of using AWS Detailed Billing.
 
+{{% notice tip %}}
+
+Already have an existing Cost and Usage Report set up? Great! You can skip this section. You'll need to provide your CUR's Report Path Prefix at a later stage in this setup.
```

Comment:
Currently, only GZIP CURs are supported. Support for ZIP CURs is coming soon enough I feel comfortable adding it here too. 

I'm thinking something like this:
```suggestion
Already have an existing Cost and Usage Report with GZIP or ZIP compression set up? Great! You can skip this section. You'll need to provide your CUR's Report Path Prefix at a later stage in this setup.
```